src/example/App/VariableText.js

```diff
@@ -52,7 +52,9 @@ const VariableText = React.createClass({
         <Collapse
           style={style.container}
           isOpened={isOpened}
-          keepCollapsedContent={keepContent}>
+          keepCollapsedContent={keepContent}
+          onHeightReady={(h) => console.log('height', h)}
```

Comment:
I was added to show the updates in the console.
But its not necessary for the PR, so I'll remove it here. :)